login.go

```diff
@@ -39,7 +40,7 @@ func (c *LoginController) Authorize(ctx *app.AuthorizeLoginContext) error {
 
 // Generate runs the authorize action.
 func (c *LoginController) Generate(ctx *app.GenerateLoginContext) error {
-	if !Development {
+	if !viper.GetBool("developer.mode.enabled") {
```

Comment:
That is exactly what I'm suggesting. I like to have all the information about a particular topic in one place with precise API's between the components. If I want to know what configuration options we support, I don't have to search through the whole codebase, I just have to look throught the config package. That we could more easily change the configuration details (for example a variable name) and even switch out viper is an added benefit. 
I try to avoid coupling like the plague.